src/lwc/edaSetupHealthCheckRun/edaSetupHealthCheckRun.js

```diff
@@ -0,0 +1,30 @@
+import { LightningElement, api } from 'lwc';
+
+// import custom labels
+import stgEDASettingsHealthCheckTitle from '@salesforce/label/c.stgEDASettingsHealthCheckTitle';
+import stgHealthCheckDescription from '@salesforce/label/c.stgHealthCheckDescription';
+import stgHealthCheckLastRun from '@salesforce/label/c.stgHealthCheckLastRun';
+import stgRunHealthCheckButton from '@salesforce/label/c.stgRunHealthCheckButton';
+
+export default class HealthCheckActionCard extends LightningElement {
+    @api runDate;
+
+    labelReference = {
+        stgEDASettingsHealthCheckTitle,
+        stgHealthCheckDescription,
+        stgHealthCheckLastRun,
+        stgRunHealthCheckButton
+    }
+
+    get lastRunDate() {
+        console.log('getting last run date: ' + this.runDate);
```

Comment:
Let's make a check to see if we have runDate and return an empty string here. Otherwise we get undefined.